src/javascript/_cookies.js

```diff
@@ -94,7 +98,7 @@
   
   var $preferencesForm = $('#dsi-cookie-form.cookies-page-dfe-sign-in__preferences-form');
 
-  $preferencesForm.on('submit', function (event) {
+  $preferencesForm.length && $preferencesForm.on('submit', function (event) {
```

Comment:
Do we need to make a null check before accessing .length?

Reply:
[jQuery selectors always returns an object](https://learn.jquery.com/using-jquery-core/selecting-elements/#does-my-selection-contain-any-elements), so `.length` assesses the existence.
#OldSchoolJS LOL 😆